Cmocka unit tests check fails in CI

##### Bug report summary
The "Unit Tests (cmocka)" CI check always fails with the following error:
```
Scanning dependencies of target valid_urls_testdriver
[ 11%] Building C object CMakeFiles/valid_urls_testdriver.dir/web/api/tests/valid_urls.c.o
[ 12%] Building C object CMakeFiles/valid_urls_testdriver.dir/web/server/web_client.c.o
make[2]: *** No rule to make target '/usr/lib/x86_64-linux-gnu/libzstd.so', needed by 'tests/valid_urls_testdriver'.  Stop.
```

##### Netdata version
netdata v1.29.3-189-g6fe27ffe0

##### Component Name
CI

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -48,7 +48,7 @@ jobs:
           ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata-all
           sudo apt-get install -y libjson-c-dev libipmimonitoring-dev libcups2-dev libsnappy-dev \
                                   libprotobuf-dev libprotoc-dev libssl-dev protobuf-compiler \
-                                  libnetfilter-acct-dev libmongoc-dev  libcmocka-dev
+                                  libnetfilter-acct-dev libmongoc-dev  libcmocka-dev libzstd-dev
       - name: Configure
         run: |
           autoreconf -ivf

